foundry.toml

```diff
@@ -0,0 +1,16 @@
+[profile.default]
+solc_version = "0.8.9"
+src = "contracts"
+test = "test"
+out = "build"
+libs = ["lib", "node_modules"]
+optimizer = true
+optimizer_runs = 1000
+auto_detect_solc = true
```

Comment:
`solc_version` is already specified, so I think it is not necessary to explicitly write `auto_detect_solc`.